app/code/Magento/Cms/Controller/Adminhtml/Wysiwyg/Directive.php

```diff
@@ -73,10 +78,14 @@ public function execute()
             /** @var Config $config */
             $config = $this->_objectManager->get(Config::class);
             $imagePath = $config->getSkinImagePlaceholderPath();
-            $image->open($imagePath);
-            $resultRaw->setHeader('Content-Type', $image->getMimeType());
-            $resultRaw->setContents($image->getImage());
-            $this->_objectManager->get(\Psr\Log\LoggerInterface::class)->critical($e);
+            try {
+                $image->open($imagePath);
+                $resultRaw->setHeader('Content-Type', $image->getMimeType());
+                $resultRaw->setContents($image->getImage());
+                $this->_objectManager->get(\Psr\Log\LoggerInterface::class)->critical($e);
+            } catch (\Exception $e) {
+                $this->_objectManager->get(\Psr\Log\LoggerInterface::class)->critical($e);
```

Comment:
I guess the expected errors should be treated with "error" or even "warning" severity instead of "critical" in both cases.
Those errors are usually thrown because of the unsupported image format or missing file etc.